File: function-center/src/main/java/cn/zcoder/xxp/function/ui/activity/AddNumActivity.java
package cn.zcoder.xxp.function.ui.activity;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;

import cn.zcoder.xxp.base.ui.activity.BaseActivity;
import cn.zcoder.xxp.function.R;
import cn.zcoder.xxp.ui.widget.ToolBar;

/**
 * Author : zhongwenpeng
 * Email : 1340751953@qq.com
 * Time :  2018/1/24
 * Description : 添加手机号码
 */

public class AddNumActivity extends BaseActivity {


    ToolBar toolBar;

    public static void startAction(Context context) {
        context.startActivity(new Intent(context, AddNumActivity.class));
    }

    @Override
    protected int getLayoutId() {
        return R.layout.activity_add_num;
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        toolBar.setTitle("添加号码")
                .setOnLeftImageListener(new ToolBar.OnLeftImageListener() {
                    @Override
                    public void onClick() {
                        finish();
                    }
                });
    }

}

File: function-center/src/main/java/cn/zcoder/xxp/function/ui/activity/AddNumActivity.kt
package cn.zcoder.xxp.function.ui.activity

import android.content.Context
import android.content.Intent
import android.os.Bundle

import cn.zcoder.xxp.base.ui.activity.BaseActivity
import cn.zcoder.xxp.function.R
import cn.zcoder.xxp.ui.widget.ToolBar

/**
 * Author : zhongwenpeng
 * Email : 1340751953@qq.com
 * Time :  2018/1/24
 * Description : 添加手机号码
 */

class AddNumActivity : BaseActivity() {



    override fun getLayoutId(): Int {
        return R.layout.activity_add_num
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        findViewById<ToolBar>(R.id.mToolBar).setTitle("添加号码")
                .setOnLeftImageListener({finish()})
    }



}
